### PR TITLE
CLC-5638 OEC: add supervisors and course_supervisors to export sheets

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/CLC-5638

### CODE REVIEW

File: app/models/oec/export_task.rb [hunk 1]
```diff
@@ -20,6 +27,15 @@ def run_internal
         add_checking_conflicts(course_instructors, locked_confirmation_row, %w(LDAP_UID COURSE_ID))
         add_checking_conflicts(instructors, locked_confirmation_row, %w(LDAP_UID))
 
+        supervisors.matching_dept_name(locked_confirmation_row['DEPT_FORM']).each do |supervisor|
+          course_supervisor_row = {
+            'COURSE_ID' => locked_confirmation_row['COURSE_ID'],
+            'LDAP_UID' => supervisor['LDAP_UID'],
+            'DEPT_NAME' => locked_confirmation_row['DEPT_FORM']
```
Comment (johncrossman) [inline app/models/oec/export_task.rb]: Is this right, @paulkerschen?  Do you want to pull DEPT_NAME?

**Comment by johncrossman** [comment]
Looks good. Waiting on one comment. Then I'll merge.

File: app/models/oec/export_task.rb (hunk 1 above)
Comment (paulkerschen) [inline app/models/oec/export_task.rb]: @johncrossman Surprisingly, it is right... this is the wonky aspects of the data that our current Blue setup is expecting.

**Comment by johncrossman** [comment]
Blessed are the wonky!

**Pull request merged**